src/wordpress-recommendations.lib.php

```diff
@@ -26,70 +25,244 @@
+                // phpcs:disable Generic.Files.LineLength
+                $recommendations['wpSaltAgeDiscriminator'] = array(
+                    __('WordPress Salt & Security Keys should be updated', 'sucuri-scanner') => __('Updating WordPress Salt & Security Keys after a compromise and on a regular basis, at least once a year, reduces the risks of session hijacking.', 'sucuri-scanner'),
+                );
+                // phpcs:enable
+            }
+        }
+
+        /*
+         * Check if there is a standard administrator/admin account.
+         * @see https://sucuri.net/guides/wordpress-security/#uac
+         */
+        if (!empty(get_users(array('role' => 'administrator', 'search' => 'admin*')))) {
+            // phpcs:disable Generic.Files.LineLength
+            $recommendations['adminBadUsername'] = array(
+                __('Admin/Administrator username still exists', 'sucuri-scanner') => __('Using an unique username and removing the default admin/administrator account make it more difficult for attackers to brute force your WordPress.', 'sucuri-scanner'),
+            );
+            // phpcs:enable
+        }
+
+        /*
+         * Check if user is using the super-admin for day-to-day operations.
+         * @see https://sucuri.net/guides/wordpress-security/#uac
+         */
+        $wpUsersCount = count_users();
+        if ($wpUsersCount['total_users'] === 1) {
+            // phpcs:disable Generic.Files.LineLength
+            $recommendations['lonelySuperAdmin'] = array(
+                __('Use super admin account only when needed', 'sucuri-scanner') => __('Create an Editor account instead of always using the super-admin to reduce the damage in case of session hijacking.', 'sucuri-scanner'),
+            );
+            // phpcs:enable
+        }
+
+        /*
+         * Check if user a 2FA plugin is being used.
+         * @see https://sucuri.net/guides/wordpress-security/#pwd
+         *
+         * First the script will get the plugins name, then run a regex match to
+         * identify possible 2FA plugins being used.
+         *
+         * NOTE: $wpPluginsInstalledName, $wpPluginsActivatedName, $wpPluginsDeactivatedName
+         * are created by this feature.
+         */
+        $wpPluginsInstalled = get_plugins();
+        foreach ($wpPluginsInstalled as $pluginPath => $pluginDetails) {
+            $wpPluginsInstalledName[] = $pluginDetails['Name'];
+            if (is_plugin_active($pluginPath)) {
+                $wpPluginsActivatedName[] = $pluginDetails['Name'];
+            } else {
+                $wpPluginsDeactivatedName[] = $pluginDetails['Name'];
+            }
+        }
+        // phpcs:disable Generic.Files.LineLength
+        $wp2faPluginsInstalled = preg_grep('/(2FA|2\sFactor|Two\sFactor|Two-Factor|2-Factor|Secure\sLogin|SecSign|Authentication|Wordfence\sSecurity|iThemes\sSecurity\sPro|Limit\sLogin|LDAP)/i', $wpPluginsInstalledName);
```

Comment:
Probably should check `$wpPluginsActivatedName` rather than `$wpPluginsInstalledName`. Same for backups.

Also, it seems impractical to try to match all 2FA plugins or all backup plugins. There's bound to be some later that won't be matched by this pattern. I guess that just means some people might see this recommendation when they shouldn't.

For backups, they might also have a cronjob, or a virtual server might have scheduled backups, or their web hosting provider might have a method of backup included in their hosting.

Reply:
Or adjust the copy to let our users known that this is not an exhaustive verification and that we may have missed another solution in place. For example: _We failed to detect some of the most popular 2fa/backups plugins. Do you have another solution in place? If not, we recommend you add one to protect your website_